### ISSUE TITLE
Different plane seg mask number compared to PlaneRCNN

### ISSUE THREAD

**Issue by mf-zhang** [EryiXie/PlaneRecNet#10]
Thank you for releasing this great work!

I found that there are fewer plane masks per image sample in your json label file compared to PlaneRCNN.

For example, in scene0000_00/color/59.jpg, there are 23 masks in PlaneRCNN and there are 7 masks in your released json file.

Is it because you filtered the small-area masks out? or is it because you used the plane masks of PlaneNet?

Thank you very much!

**Comment by EryiXie** [comment]
Yes, I filtered out very small plane areas and did not put them inside the annotation file. In the implementation of PlaneRCNN, small planes are filtered out while reading masks annotation in the dataset script (as the authors mentioned in their paper), and I feel it is not efficient to do it online, since we can delete them all at once in the annotation file.

**Comment by shuimushan** [comment]
> Yes, I filtered out very small plane areas and did not put them inside the annotation file. In the implementation of PlaneRCNN, small planes are filtered out while reading masks annotation in the dataset script (as the authors mentioned in their paper), and I feel it is not efficient to do it online, since we can delete them all at once in the annotation file.

Could you please release your annotation script? It will help me a lot.
